package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "floating-tag-action",
+  "version": "0.1.0",
+  "description": "Ensures that all tags in the repository in the current path have corresponding major version tags.",
+  "main": "index.ts",
+  "scripts": {
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+    "dist": "ncc build -o dist index.ts"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/giantswarm/install-binary-action.git"
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/giantswarm/floating-tag-action/issues"
+  },
+  "homepage": "https://github.com/kopiczko/my-js-action#readme",
+  "dependencies": {
+    "@actions/core": "^1.2.7",
+    "@actions/exec": "^1.0.4",
+    "@actions/tool-cache": "^1.6.1",
+    "@types/semver": "^7.3.5",
```

Comment:
I thought when something is in `dependences` then it is in `devDependences` too. Am I wrong?

Reply:
That is correct, I was a bit unclear with my wording.
What I meant was that the `@types/semver` package is only necessary during development (for code editor integration and for typechecking when the code is compiled to JS), and not at runtime, so it should be listed in `devDependencies`, instead of `dependencies`

Reply:
Got it, thanks!